modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -45,43 +62,32 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: thanos-token
+  namespace: openshift-ingress-operator
   annotations:
     kubernetes.io/service-account.name: thanos
   type: kubernetes.io/service-account-token
 EOF
 ----
 
-. Define a `TriggerAuthentication` object within the `openshift-ingress-operator` namespace using the service account's token.
-
-.. Define the variable `secret` that contains the secret by running the following command:
-+
-[source,terminal]
-----
-$ secret=$(oc get secret | grep thanos-token | head -n 1 | awk '{ print $1 }')
-----
+. Define a `TriggerAuthentication` object within the `openshift-ingress-operator` namespace by using the service account's token.
 
 .. Create the `TriggerAuthentication` object and pass the value of the `secret` variable to the `TOKEN` parameter:
 +
 [source,terminal]
 ----
-$ oc process TOKEN="$secret" -f - <<EOF | oc apply -f -
-apiVersion: template.openshift.io/v1
-kind: Template
-parameters:
-- name: TOKEN
-objects:
-- apiVersion: keda.sh/v1alpha1
-  kind: TriggerAuthentication
-  metadata:
-    name: keda-trigger-auth-prometheus
-  spec:
-    secretTargetRef:
-    - parameter: bearerToken
-      name: \${TOKEN}
-      key: token
-    - parameter: ca
-      name: \${TOKEN}
-      key: ca.crt
+$ oc apply -f - <<EOF
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: keda-trigger-auth-prometheus
```

Comment:
One more:
```suggestion
  name: keda-trigger-auth-prometheus
  namespace: openshift-ingress-operator
```

Reply:
Thanks, Grant. I'll ask Shudi to complete a review of this PR.